### PR TITLE
[KP-714] Update config.json to contain descriptions, placeholders, defaults, learnURLs.

### PR DESCRIPTION
**Pull request by kenskyfish**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Kaholo/kaholo-plugin-textEditor.git"
   },
   "author": "Kaholo",
-  "license": "ISC"
-}
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Kaholo/kaholo-plugin-template/issues"
```
Comment (graddaniel) [inline package.json]: Outdated urls

File: textEditorService.js [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
-        return returnContent ? await fs.readFileSync(this.fixedPath, 'utf8') : `File ${this.path} updated.`;
+  async appendToFile({ content, createIfNeeded, returnContent }) {
+    if (!fs.existsSync(this.fixedPath)) {
+      if (createIfNeeded) {
+        const result = await this.createFile({ content });
+        return returnContent ? fs.promises.readFile(this.fixedPath, "utf8") : result;
+      }
+      throw new Error(`File ${this.path} doesn't exist!`);
     }
+    await fs.appendFileSync(this.fixedPath, content);
```
Comment (graddaniel) [inline textEditorService.js]: Why no try/catch around that one?

**Review by graddaniel — CHANGES_REQUESTED** [review]

File: textEditorService.js (hunk 2 above)
Comment (kenskyfish) [reply textEditorService.js]: Because I lack the know-how to do that in a way that eslint is OK with. Care to tutor me?

File: package.json (hunk 1 above)
Comment (kenskyfish) [reply package.json]: done - https://github.com/Kaholo/kaholo-plugin-textEditor/issues

File: textEditorService.js (hunk 2 above)
Comment (graddaniel) [reply textEditorService.js]:
```suggestion
        try { 
            await fs.appendFileSync(this.fixedPath, content);
        }
        catch (error) {
            throw new Error(`Error updating file ${this.path}: ${error.message || JSON.stringify(error)}`);
        }
```
What about this one?

**Review by graddaniel — CHANGES_REQUESTED** [review]

**Review by graddaniel — APPROVED** [review]

**Pull request merged**